Getting Started with Splitter (#6266)
DevExpress/devextreme-documentation: concepts/05 UI Components/Splitter/00 Getting Started with Splitter/05 Build a Layout.md

> In this tutorial, we create a component that splits only the part of the page. To achieve this functionality, we set Splitter [width](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#width) and [height](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#height) properties to 500 and 400px respectively.

Reviewer: We don't use the word "we" in this sense. Documentation can say that "we recommend" something (we = DevExpress) or "we implemented a new capability". But when it's a tutorial, it's our customers who goes through those steps, so we generally say "you" or we just refer to the tutorial. 

"achieve functionality" - those two words aren't really used together like that. You can maybe "implement functionality" or "achieve a goal".

 ```suggestion
In this tutorial, the Splitter takes up only a part of the page. To achieve this goal, set the component's [width](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#width) and [height](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#height) properties to 500px and 400px respectively.
```

> [note] If you want to split the entire page, place a Splitter in the `<body>` tag. Set the width and height options to 100% for both the Splitter and the `<html>` tag.

Reviewer 2: ```suggestion
[note] If you want to split the entire page, place a Splitter in the `<body>` tag and set the width and height options to 100% for both the Splitter and the `<html>` tag.
```

> [note] If you want to split the entire page, place Splitter in the `<body>` tag. Set the width and height options to 100%. Then set the `<html>` tag width and height styles to 100% too.

Reviewer: You need to create a new object (a Splitter). 

An example could be like this: "If you want to get a driver's license, you need to learn to drive a car." Although it can be any car... or you may think you refer to cars in general, it's still "a car". You cannot "drive car". When you're doing something with an object, it's hard to think of an example when you wouldn't need an article. 

You could've said "DxSplitter allows you to create nice layouts". In that case, you don't need an article. But even with that name, if you do something with an object, you would need an article: "place a DxSplitter in the body tag".

And in the last sentence, you can mention both the splitter and the <html> tag. This change makes everything a bit more compact. 

```suggestion
[note] If you want to split the entire page, place a Splitter in the `<body>` tag. Set the width and height options to 100% for both the Splitter and the `<html>` tag.
```

> Splitter can be [orientated](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation) either horizontally (default) or vertically.

Reviewer: In any event, "orientate" doesn't sound right, at least in American English. It should be "oriented" instead of "orientated". But I want to make it more about how items are arranged, than about the "splitter orientation". 

```suggestion
A Splitter can [arrange its items](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation) either horizontally (default) or vertically.
```

> You can specify the Splitter item layout through two options:

Reviewer: I think the previous option was related to "layout". What you talk about in the list is more related to "content". 

```suggestion
You can use two options to specify Splitter item content:
```

- [items](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/)    

> Specify panes explicitly in the code. This option is used in this tutorial.

Reviewer: ```suggestion
Specifies pane content directly in code. This option is used in this tutorial.
```

- [dataSource](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#dataSource)    

> You can predefine pane structure and store it separately from the main code.

Reviewer: ```suggestion
Obtains pane content stored separately from the main code.
```

> Each Splitter item can hold another [splitter](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#splitter). The nested splitter uses the same configuration properties as the main one.

Reviewer: I'm not sure why some Splitters are capitalized and others aren't. 

And the word "splitter" is used multiple times in these sentences anyway. So it's better to use "pane" instead of "Splitter item" - cut down on repetition.

```suggestion
Each pane can hold another [Splitter](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#splitter). Each nested Splitter uses the same configuration properties as its parent.
```

> Additionally, you can specify the [size of a separator bar](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#separatorSize) in pixels.

Reviewer: I don't like the start of this sentence - "Additionally". 

We talk about the layout - panes, their orientation, their content, nested structures. And then, all of a sudden, "oh I forgot to mention one more thing". It comes out as being unrelated to everything you said previously. 

One way to deal with that is by using headers in this topic - then you can separate your thoughts. 

Another way is to try and link what you're saying to what you've already said. That way, it would seem like there's logic to all this text: 

```suggestion
Once you set up pane layout, the Splitter displays separator bars between panes. You can specify the [width of these bars](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#separatorSize) in pixels.
```
